actions/update_address_group.yaml

```diff
@@ -1,4 +1,4 @@
-name: update_address_object
+name: update_address_group_object
```

Comment:
Indeed, there is another action with the same name 

https://github.com/StackStorm-Exchange/stackstorm-fortinet/blob/01d477824b4dbf5a635d028a2663ae87f086a314/actions/update_address_object.yaml#L1-L4